bpf/bpf_lxc.c

```diff
@@ -679,6 +679,34 @@ static __always_inline int handle_ipv4_from_lxc(struct __ctx_buff *ctx,
 		}
 	}
 
+#ifdef ENABLE_EGRESS_GATEWAY
```

Comment:
@anfernee Could you also update `bpf/Makefile` to add this macro to `LXC_OPTIONS` and `MAX_LXC_OPTIONS`? That will enable compilation and complexity tests.